README.md

```diff
@@ -1,2 +1,80 @@
-# ai_on_edge_with_fiware
-This is a project to implement an AI on Edge application with FIWARE. 
+# Context Broker environment setup
```

Comment:
I think it makes more sense to share here the whole architecture. 
But only via the commits the reader will understand that we need first to set up CB and Mongo, then the rest of the components. 
=> correction: 
1. Start with the overall architecture. 
2. Add a sub-title context broker confirguration: add there the command of the POST and refer to the data model 
3. then with each new PR, we add sub titles of the other components.